tests/unit/amo/reducers/test_addonsByAuthors.js

```diff
@@ -854,31 +836,31 @@ describe(__filename, () => {
         undefined,
         fetchAddonsByAuthors({
           ...params,
-          authorUsernames: ['someOtherAuthor'],
+          authorIds: [randomAuthorId1],
         }),
       );
 
-      expect(getLoadingForAuthorNames(state, ['author2'])).toEqual(null);
+      expect(getLoadingForAuthorIds(state, [randomAuthorId2])).toEqual(null);
     });
 
-    it('returns null when no authorUsernames provided', () => {
+    it('returns null when no authorIds provided', () => {
       const state = reducer(undefined, fetchAddonsByAuthors(params));
 
-      expect(getLoadingForAuthorNames(state, [])).toEqual(null);
+      expect(getLoadingForAuthorIds(state, [])).toEqual(null);
     });
   });
 
-  describe('getCountForAuthorNames', () => {
+  describe('getCountForAuthorIds', () => {
     const params = {
       addons: [],
-      authorUsernames: ['author1'],
+      authorIds: [randomAuthorId1],
       count: 0,
       errorHandlerId: 'error-handler-id',
       pageSize: EXTENSIONS_BY_AUTHORS_PAGE_SIZE,
     };
 
-    it('returns count for just authorUsernames', () => {
-      const count = 123;
+    it('returns count for just authorIds', () => {
+      const count = randomAuthorId1;
```

Comment:
This feels weird. I know it works, but it makes it seem like there's a relationship between the authorId passed in and the count, which there is not. I think it would be better to just set `count` to a number.

Reply:
oops, sorry about that. Search & replace false positives... I should have caught that, sorry 😞